CityOfDetroit/bloom
# Fetch UnitsSummary in listings query

## Issue

- Closes #239 

## Description

Populates UnitsSummary in the data returned by the listing query.


## Type of change


## How Can This Be Tested/Reviewed?

Start the backend and visit the backend /listings endpoint and verify that the `unitsSummary` prop is populated for the listings.

## Checklist:
